SCClassLibrary/Common/UnitTesting/UnitTest.sc

```diff
@@ -189,6 +189,22 @@ UnitTest {
 		}
 	}
 
+	testForException { |func, errorClass|
+		var test = if(errorClass.isNil) { _.notNil } { _.isKindOf(errorClass.asClass) };
+		var result = false;
+		func.try { |error| result = test.(error) };
+		^result
+	}
+
+	assertException { | func, message, errorClass, report = true, onFailure, details |
```

Comment:
`errorClass` should come before `message`, this matches the argument ordering of `assertEquals`.

Reply:
yes, if we make it mandatory, that would be ok.